modules/local/samplesheet_check.nf

```diff
@@ -13,11 +13,9 @@ process SAMPLESHEET_CHECK {
     path '*.csv'       , emit: csv
     path "versions.yml", emit: versions
 
-    script: // This script is bundled with the pipeline, in nf-core/taxprofiler/bin/
+    script: // detect_reads.py script is bundled with the pipeline, in nf-core/taxprofiler/bin/
     """
-    check_samplesheet.py \\
-        $samplesheet \\
-        samplesheet.valid.csv
+    python3 $projectDir/bin/detect_reads.py $samplesheet samplesheet_validated.csv
```

Comment:
```suggestion
    python3 $projectDir/bin/detect_reads.py \\
        $samplesheet \\
        samplesheet_validated.csv
```

Will probably try and replace this to groovy in a second PR so will format it nicely for now!